geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeEmbeddedPolicy.java

```diff
@@ -126,23 +136,23 @@ static GeodeEmbeddedPolicy create() {
       this.refCount = new AtomicInteger();
     }
 
-    @Override GeodeEmbeddedPolicy share() {
```

Comment:
@XuQianJin-Stars , please refrain from moving unrelated code in the future. You have moved `share()` to the end of the class, and it did make it harder to understand the diff. It is lot a huge issue, but it is something that makes a difference between "ok, I see this PR is good, I will merge it immediately" and "well, I need to load it to my IDE and inspect carefully".

Please compare with https://github.com/apache/calcite/commit/484b112641a1a9780bafc43cda6079ebc4806ca3#diff-6820bea7216536ddc41a040817f66218R135-R145 where the diff much simpler

Note: you did remove `synchronized` modifier from `before()` method, and it is very hard to spot. The modifier was important.